matrix-sdk-android/src/main/java/im/vector/matrix/android/api/auth/data/Credentials.kt

```diff
@@ -45,7 +45,7 @@ data class Credentials(
          * @Deprecated. Clients should extract the server_name from user_id (by splitting at the first colon)
          * if they require it. Note also that homeserver is not spelt this way.
          */
-        @Json(name = "home_server") val homeServer: String,
+        @Json(name = "home_server") val homeServer: String?,
```

Comment:
Is the server returning null value? Or is it just absent from response?

Reply:
I do not know.